src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -151,16 +151,16 @@ key *docker-compose-options-split-regex*. If you supply e.g. "!SPLIT!" as value,
 
 === Docker_File task
 
-The basic behavior of a Docker_File task is to create an image with Docker build and to create a container based on the built image. At the end of the job, the created image and container are deleted. Advanced options will allow to parametrize these actions.
+The basic behavior of a Docker_File task is to build an image and run a container using the built image. At the end of the task (the dockerfile execution), the container is stooped and the image is deleted. 
+
+Advanced options will allow to parametrize these actions.
 
 In order for Docker_File tasks to work, the _Node_ must have Docker installed. Please refer to the official
- Docker documentation to see how to install http://docs.docker.com/engine/installation/[Docker^]
+ Docker documentation to see how to install http://docs.docker.com/engine/installation/[Docker^].
 
-A Docker_File task expects the content of a Dockerfile inside the _Script_ section. You can find out how to write Dockerfile
+To use a Docker_File task, put the content of a Dockerfile inside the _Task Implementation_ section. You can find out how to write Dockerfile
 in the official https://docs.docker.com/engine/reference/builder/[Dockerfile documentation^].
-
-
-The Docker_File task will run a combination of docker commands to build, start, stop and remove containers. 
+When the Docker_File task is launched, a succession of docker commands is to build image, run container, stop container and remove image. 
```

Comment:
A Docker_File task allows executing a succession of docker commands according to lifecycle of Docker containers. In order, `docker build`, `docker run`, `docker stop`, and `docker rmi` are ran when a Docker_File task is executed.